lib/Newsletter/Renderer/EscapeHelper.php

```diff
@@ -42,7 +42,7 @@ static function unescapeHtmlStyleAttr($string) {
    */
   static function escapeHtmlLinkAttr($string) {
     $string = self::escapeHtmlAttr($string);
-    if (preg_match('~^javascript:|^data:text|^data:application~i', $string) === 1) {
+    if (preg_match('/(\s)*(javascript:|data:text|data:application)+/ui', $string) === 1) {
```

Comment:
Why the `+` after the right group? Also, the `(\s)*` can be just `\s*`.

Reply:
Just leftovers after testing. I'll remove that.